### PR TITLE
Story 15

### LINKED ISSUE #106: User Story 15, Items Index Page Statistics
```
As any kind of user on the system
When I visit the items index page ("/items")
I see an area with statistics:
- the top 5 most popular items by quantity purchased, plus the quantity bought
- the bottom 5 least popular items, plus the quantity bought

"Popularity" is determined by total quantity of that item fulfilled
```

DO THESE STORIES FIRST:
- User Story 14, Items Index Page

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #108

child of #125

depends on #107

### PR DESCRIPTION
**Pull request by cnodland**
closes #106

### CODE REVIEW

**Review by wipegup — APPROVED** [review]
Looks Good, Nice Testing

File: app/models/item.rb [hunk 1]
```diff
@@ -15,4 +15,20 @@ class Item < ApplicationRecord
...
+    .group(:id)
+    .order("item_count #{order}")
+    .limit(5)
+  end
+
+  def total_sold
+    order_items.where("order_items.fulfilled = ?", true)
+               .pluck("sum(order_items.quantity)as sold_count")
+               .first
+  end
```
Comment (whois-jon-peterson) [inline app/models/item.rb:32]: This query could also be expressed as `where(...).sum('order_items.quantity')` which might be a little more concise.

@tnodland @wipegup thoughts?

**Review by whois-jon-peterson — APPROVED** [review]

**Review by wipegup — APPROVED** [review]
Good Change For making the total sold more concise
